play/src/i18n/en-US/chat.ts

```diff
@@ -42,10 +42,11 @@ const chat: BaseTranslation = {
     searchUser: "Search for user, map, etc ...",
     searchChat: "Search for channel, message, etc ...",
     people: "People",
-    rooms: "Rooms",
+    rooms: "Your Discussions",
```

Comment:
Use "my"?
Here we talk about "Discussion" and in another part, we talk about "room". I think that we need to do a choice.